packages/firebase_auth/example/lib/main.dart

```diff
@@ -37,6 +37,10 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   Future<String> _message = new Future<String>.value('');
+  TextEditingController smsCodeController = new TextEditingController();
```

Comment:
nit: add underscore to variables like `_message`

Reply:
Done